tests/model_generator/test_model_pydantic_generator.py

```diff
@@ -185,14 +186,18 @@ class Meta:
 
 
 def test_decimal_field(schema_with_decimal_field: types.JsonDict) -> None:
-    expected_result = """
+    expected_result = f"""
 from dataclasses_avroschema import types
 from dataclasses_avroschema.pydantic import AvroBaseModel
 
 
 
 class Demo(AvroBaseModel):
     foo: types.condecimal(max_digits=10, decimal_places=3)
+
+    {templates.METACLASS_DECORATOR}
```

Comment:
`{templates.METACLASS_DECORATOR}` is not needed. It is used only when `enums` has a `class Meta` (depending on the python version)

Reply:
Agreed!